김현승/reco-admin-api.v2.yaml

```
@@ -0,0 +1,284 @@
+openapi: 3.0.3
+info:
+  title: Recommendation & Admin Domain API - Compass Travel
+  description: |
+    # Recommendation & Admin 도메인 API 명세서 (v2.0)
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)